### ISSUE TITLE
ePub?

### ISSUE THREAD

**Issue by mgajda** [shi-yan/WebGPUTutorial#2]
It would be great to see the book as ePub.

**Comment by shi-yan** [comment]
Yes, I used to have both epub and pdf generators for this book. but they are out of sync with the recent development. I will bring them back when I have time.
